src/cljs/main/broadfcui/page/workspace/notebooks/notebooks.cljs

```diff
@@ -327,7 +330,23 @@
                            :reload-after-delete #(this :-get-clusters-list-if-whitelisted))]))]))
    :component-did-mount
    (fn [{:keys [this]}]
-     (this :-is-leo-whitelisted))
+     (do (this :-is-leo-whitelisted)
+         (.addEventListener js/window "message" (this :-handle-postmessage-event))))
+
+   :component-will-unmount
+   (fn [{:keys [this]}]
+     (.removeEventListener js/window "message" (this :-handle-postmessage-event)))
+
+   ; Communicates with the Leo notebook extension
+   :-handle-postmessage-event
+   #(fn [e]
```

Comment:
Is this correct -- i.e. will `(this :-handle-postmessage-event)` return the same function reference for both `addEventListener` and `removeEventListener`?